cli/daemon/dash/server.go

```diff
@@ -27,17 +28,23 @@ var upgrader = websocket.Upgrader{
 var assets embed.FS
 
 // NewServer starts a new server and returns it.
-func NewServer(runMgr *run.Manager, tr *trace.Store) *Server {
+func NewServer(runMgr *run.Manager, tr trace2.Store) *Server {
 	assets, err := fs.Sub(assets, "dashapp/dist")
 	if err != nil {
 		log.Fatal().Err(err).Msg("could not get dash assets")
 	}
 
+	proxy, err := dashproxy.New("https://devdash.encore.dev")
```

Comment:
Could we add a `var DevDashURL string = "https://devdash.encore.dev"` as a package level variable - such that during dev builds we can change this to point at a local `npm run dev` version of the dev dash.

I suggest we also add a `var CacheDevDash bool` as well, so we can disable caching